### PR TITLE
Added translate: zh-cn

### PR DESCRIPTION
**Pull request by penghao123456**
Added translate `zh-cn` in 5 commits.

Some key does not translated because I don't know where they used.

### CODE REVIEW

File: addons/dialogue_manager/l10n/zh-cn.po [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+msgstr "关闭其他文件"
+
+msgid "buffer.copy_file_path"
+msgstr "复制文件路径"
+
+msgid "buffer.show_in_filesystem"
+msgstr "在 Godot 侧边栏中显示"
+
+msgid "settings.revert_to_default_test_scene"
+msgstr ""
```
Comment (nathanhoad) [inline addons/dialogue_manager/l10n/zh-cn.po:130]: This one shows up in the settings window when you specify a custom test scene:

image

File: addons/dialogue_manager/l10n/zh-cn.po [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+msgstr "ID 不存在。"
+
+msgid "errors.invalid_indentation"
+msgstr "缩进非法。"
+
+msgid "errors.condition_has_no_content"
+msgstr "条件下方不能为空。"
+
+msgid "errors.incomplete_expression"
+msgstr ""
```
Comment (nathanhoad) [inline addons/dialogue_manager/l10n/zh-cn.po:262]: This one is when you have an expression that doesn't quite look right:

image

File: addons/dialogue_manager/l10n/zh-cn.po [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+
+msgid "errors.invalid_expression_for_value"
+msgstr ""
```
Comment (nathanhoad) [inline addons/dialogue_manager/l10n/zh-cn.po:265]: This one doesn't appear to be used anywhere so its safe to ignore.

File: addons/dialogue_manager/l10n/zh-cn.po [hunk 4]
```diff
@@ -0,0 +1,403 @@
...
+msgstr "句号不应存在。"
+
+msgid "errors.unexpected_boolean"
+msgstr "true/false 不应存在。"
+
+msgid "errors.unexpected_string"
+msgstr ""
+
+msgid "errors.unexpected_number"
+msgstr ""
```
Comment (nathanhoad) [inline addons/dialogue_manager/l10n/zh-cn.po:304]: This is when you do something like forget the brackets around a function arguments:

image

File: addons/dialogue_manager/l10n/zh-cn.po [hunk 5]
```diff
@@ -0,0 +1,403 @@
...
+
+msgid "errors.unexpected_variable"
+msgstr ""
```
Comment (nathanhoad) [inline addons/dialogue_manager/l10n/zh-cn.po:307]: This is when you do something like forget the brackets around a function arguments:

image

File: addons/dialogue_manager/l10n/zh-cn.po [hunk 6]
```diff
@@ -0,0 +1,403 @@
...
+
+msgid "errors.invalid_index"
+msgstr "索引无效。"
+
+msgid "errors.unexpected_assignment"
+msgstr ""
```
Comment (nathanhoad) [inline addons/dialogue_manager/l10n/zh-cn.po:313]: This is when you use a single `=` in an `if` line:

image

**Review by nathanhoad — COMMENTED** [review]
Thank you so much! I've added some comments to help give you some context for the missing ones.

File: addons/dialogue_manager/l10n/zh-cn.po (hunk 3 above)
Comment (penghao123456) [reply addons/dialogue_manager/l10n/zh-cn.po:265]: Ok. I will let it blank in my pull request.

**Comment by penghao123456** [comment]
I had added the translate in the pull request.

**Pull request merged**